pkg/kudoctl/util/repo/repo_operator.go

```diff
@@ -117,13 +120,26 @@ func (r *OperatorRepository) GetPackageReader(name string, version string) (io.R
 
 // GetPackageBundle provides an Bundle for a provided package name and optional version
 func (r *OperatorRepository) GetPackageBundle(name string, version string) (Bundle, error) {
```

Comment:
I feel like this should not be part of repo 🤔 I thought we have several ways how to obtain packages:
- local
- tarball
- url
- repo

I think all those should branch from the same place ideally? Since obtaining packages from http has nothing to do with repository I think the code should live somewhere else.

Reply:
yeah... I went back and forth on the same thoughts